make/mt-config/mt-presets.mk

```diff
@@ -105,6 +106,13 @@ else ifeq ($(PRESET), STM32F4)
     XI_BSP_PLATFORM = stm32f4
     XI_TARGET_PLATFORM = stm32f4
 
+# ST Micro STM32F4_NUCLEO_WIFI
+else ifeq ($(PRESET), STM32F4_NUCLEO_WIFI)
+    CONFIG = $(CONFIG_STM32F4_NUCLEO_WIFI)
+    TARGET = $(TARGET_STATIC_DEV)
```

Comment:
I think the default should be TARGET_STATIC_REL to make customer's and Travis's builds default to a release version.